tools/ci/testlist/arm-05.dat

```diff
@@ -26,15 +26,15 @@ CMake,nrf52840-dk:buttons
 CMake,nrf52840-dk:cdcacm
 # CMake,nrf52840-dk:composite
 CMake,nrf52840-dk:highpri
-# CMake,nrf52840-dk:mcuboot_app
-# CMake,nrf52840-dk:mcuboot_loader
+CMake,nrf52840-dk:mcuboot_app
+CMake,nrf52840-dk:mcuboot_loader
 CMake,nrf52840-dk:nsh
 CMake,nrf52840-dk:ostest_tickless
 CMake,nrf52840-dk:pwm
 CMake,nrf52840-dk:qspi
-#CMake,nrf52840-dk:rndis
-# CMake,nrf52840-dk:sdc
-# CMake,nrf52840-dk:sdc_nimble
+# CMake,nrf52840-dk:rndis
```

Comment:
can we remove all comment like nrf52840-dk:rndis.

Reply:
rndis and composite configurations need `argtable3` which is not yet ported

Reply:
Hi, @raiden00pl have you doing cmake porting of argtable3, if not in your plan, I will complete argtable3 porting soon

Reply:
@raiden00pl Please review PR https://github.com/apache/nuttx-apps/pull/1848

Reply:
@anchao great, I'll enable rndis configurations after  merging apache/nuttx-apps#1848